lntest/node.go

```diff
@@ -717,6 +720,7 @@ func (hn *HarnessNode) stop() error {
 	hn.processExit = nil
 	hn.LightningClient = nil
 	hn.WalletUnlockerClient = nil
+	hn.WatchtowerClient = nil
```

Comment:
Didn't realize this was a thing. Should we also do this for `RouterClient` and `WalletKitClient`?